### ISSUE TITLE
Surprising behavior when `Var` value does not change

### ISSUE THREAD

**Issue by netvl** [lihaoyi/scala.rx#29]
I would expect that updating a `Var` to a value equal to its previous value won't trigger `Obs` depending on this `Var`. However:

``` scala
val v = Var(10)
val o = v.foreach(println)  // prints 10
v() = 11  // prints 11
v() = 11  // again prints 11!
v() = 11  // and again
```

When `Var` is pulled through an `Rx` it works correctly:

``` scala
val v = Var(10)
val w = Rx { v() }
val o = w.foreach(println)  // prints 10
v() = 11  // prints 11
v() = 11  // nothing, just as expected
v() = 11  // again, nothing
```

**Comment by Voltir** [comment]
This is fixed as of version 0.3.0.

**Comment by LogicalTime** [comment]
If this is fixed shall we close the issue?
